docker-image-watch

```diff
@@ -104,7 +119,18 @@ if [ -f "$STAMP_FILE" ]; then
 fi
 NEW_IMAGE_DIGEST=$(get_image_digest $DOCKER_TOKEN $DOCKER_REPOSITORY $DOCKER_IMAGE $DOCKER_IMAGE_TAG)
 
+PULLED=0
+RESTARTED=0
+
 if [ "$OLD_IMAGE_DIGEST" != "$NEW_IMAGE_DIGEST" ]; then
-	send_email "$RECIPIENT" $DOCKER_REPOSITORY $DOCKER_IMAGE $DOCKER_IMAGE_TAG
+	if [ "$AUTOPULL" -eq 1 ] ; then
+		$DOCKER_CMD pull $IMAGE
+		PULLED=1
+		if [ ! -z "$RESTART_CMD" ] ; then
```

Comment:
```suggestion
		if [ -n "$RESTART_CMD" ] ; then
```

Reply:
Funny, for some reason I thought they are not equivalent (i.e., `-n` only tests whether it's defined, but doesn't check whether it's empty). That's wrong though, thanks for the suggestion.